### PR TITLE
Session 2 - products intro

### PR DESCRIPTION
**Pull request by eefaye**
Looking for content feedback; still pondering if the structure of this page can be further optimised.

index.rst has been updated for file name change. 

I intend to embed some diagrams/figures/pictures to make this page more palatable, waiting on initial feedback.

Anyone welcome to review.

### CODE REVIEW

**Comment by jcrattz** [comment]
@eefaye 

Here are my suggestions:

1. In the **Spatial resolution** bullet point, change "is generally physically limited" to "resolution is limited".
1. In the **Temporal resolution** bullet point, change "Polar orbits occur where satellites fly from pole to pole, passing over different parts of the Earth on each flyover." to "Satellites in polar orbits pass over the poles of the Earth on each orbit, but each orbit also passes over different areas than the prior orbit."
1. In the **Spectral resolution** bullet point, change "wavelength range they cover — for example, red band, green band, near-infrared band" to "wavelength range they cover, such as "red", or "green", or "near-infrared"."
1. In the "Satellite product spotlight: Landsat 8" section, in "Landsat 8 is the eight iteration", change "eight" to "eighth" or "latest".
1. In the "Satellite product spotlight: Sentinel-2" section, do we need the statement "The satellites are purposefully spaced"? Does this refers to their locations being opposite in their shared orbit? I think we should either omit this for simplicity or rephrase it to be more specific.
1. In the "Derived products" section, I think we should note the names of the fractional cover and WOfS products in DE Africa since we also note the names of the Landsat 8 and Sentinel-2 products.

**Comment by eefaye** [comment]
Thanks for the feedback being super clear and easy to action. I have made all the suggested changes but will merge after I have embedded a few images.
